provision/opensuse/etcd.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -eu
+
+sudo sed -i 's/ETCD_DATA_DIR=.*$/ETCD_DATA_DIR=\/var\/etcd\/cilium/g' /etc/sysconfig/etcd
```

Comment:
in sed you can use other characters for search in replace so you don't need to escape `/`, for example:
```
s/ETCD_DATA_DIR=.*$/ETCD_DATA_DIR=\/var\/etcd\/cilium/g'
```
can be
```
s+ETCD_DATA_DIR=.*$+ETCD_DATA_DIR=/var/etcd/cilium+g'
```

Reply:
done